packages/clarity-native-bin/package.json

```diff
@@ -33,7 +33,8 @@
     "lint": "tslint -p tsconfig.json 'src/**/*.ts' 'test/**/*.ts'",
     "lint-fix": "tslint --fix -p tsconfig.json 'src/**/*.ts' 'test/**/*.ts'",
     "test": "npm run build && nyc mocha",
-    "postinstall": "node postInstallScript.js"
+    "postinstall": "node postInstallScript.js",
+    "docs": "yarn typedoc --out docs --plugin typedoc-plugin-markdown src"
```

Comment:
Yarn is not currently a build requirement or the package manager used in this repo. Should be changed to `npm` (or the whole repo should be switched to yarn).

Reply:
changed to npm